lib/radbeacon/le_scanner.rb

```diff
@@ -1,10 +1,11 @@
 module Radbeacon
 
   class LeScanner
-    attr_accessor :duration
+    attr_accessor :duration, :options
 
     def initialize(duration = 5)
       @duration = duration
+      @options = {}
```

Comment:
So you're saying nix the options hash and just add `mac_filter` and `hcitool_duration` as attributes themselves?  

To be clear, `hcitool_duration` and `duration` are not the same.  `duration` is how many seconds a `radbeacon.scan` will last (default value is 5).  `hcitool_duration` is a boolean that determines if the custom `hcitool` binary that has been modified to include a duration option is used. 

If it's false, the regular `hcitool lescan` command is used, which has to be spawned and terminated in ruby after the `duration` timeout is reached (otherwise it will run forever).  If it's true, the scan will pass the `duration` to the custom `hcitool` binary:

```
sudo hcitool lescan --duration 5
```

which has been modified with a non-blocking socket read and a timeout that will exit peacefully on its own after the duration limit is reached.  

The second way has proven to be more stable, but requires the custom hcitool binary.  If you enable the `hcitool_duration` option without it the scan will fail.

Maybe I should rename that option so its purpose is more clear?